sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -7,7 +7,7 @@
 import * as coreHttp from '@azure/core-http';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { PipelineOptions } from '@azure/core-http';
-import { PollerLike } from '@azure/core-lro';
+import { PollerLike as PollerLike_2 } from '@azure/core-lro';
```

Comment:
This looks a bit fishy

Reply:
This just means that I need to move the PollerLike change up to core-lro. Should I change the name of the custom PollerLike? It will be just one PollerLike when that other PR is merged and the package is released (I haven't made the PR just yet).

Reply:
I wouldn't use the same name, though, as this type is getting exposed to the user

Reply:
What name should I use? I don't like any of the names that come to mind.

Reply:
PollerLike_2 feels like we made a mistake in the renaming somewhere.

You could use something like CorePollerLike for the one coming from core-lro

Reply:
Even if I try to rename all use cases of the core PollerLike (PollerLike_2) to CorePollerLike, the review file isn't changed. Probably since I'm not exporting this type

Reply:
I'm exporting CorePollerLike now!